examples/python/src/hello-world.py

```diff
@@ -0,0 +1,212 @@
+                            # was called from
+                            'expand': {
+                                'app': 'all',
+                                # if you want to expand team & channel, ensure that bot is added to the team & channel
+                                # else command will fail to expand the context
+                                # 'team': 'all',
+                                # 'channel': 'all',
+                            },
+                        },
+                    }
+                ],
+            },
+            {
+                # binding for a command, command with embedded form
+                'location': '/command',
+                'bindings': [
+                    {
+                        'description': 'test command',
+                        'hint': '[This is testing command]',
+                        # this will be the command displayed to user as /second-command
+                        'label': 'second-command',
+                        'icon': 'icon.png',
+                        'bindings': [
+                            {
+                                'location': 'send',
+                                'label': 'send',
+                                'form': FORM
+                            },
+                        ],
+                    }
+                ],
+            },
+            {
+                'location': '/channel_header',
+                'bindings': [
+                    {
+                        'location': 'send-button',
+                        'icon': 'icon.png',
+                        'label': 'send hello message',
+                        'form': FORM,
+                    },
+                ],
+            },
+        ],
+    }
+
+
+@app.route('/ping', methods=['POST'])
+def on_ping() -> dict:
+    logging.debug('ping...')
+    return {'type': 'ok'}
+
+
+@app.route('/install', methods=['GET', 'POST'])
+def on_install() -> dict:
+    print(f'on_install called with payload , {request.args}, {request.data}', flush=True)
+    _subscribe_team_join(request.json['context'])
+    return {'type': 'ok', 'data': []}
+
+
+@app.route('/first_command', methods=['POST'])
+def on_first_command():
+    print(f'/first_command called ')
+    response_message = 'Hello! response from /first_command'
+    return {'type': 'ok', 'text': response_message}
+
+
+@app.route('/bot_joined_team', methods=['GET', 'POST'])
+def on_bot_joined_team() -> dict:
+    context = request.json['context']
+    logging.info(
+        f'bot_joined_team event received for site:{context["mattermost_site_url"]}, '
+        f'team:{context["team"]["id"]} name:{context["team"]["name"]} '
+        f'{request.args} {request.data}'
+    )
+    # Here one can subscribe to channel_joined/left events as these required team_id now to be subscribed,
+    # hence use the team_id received in the event and make a call for subscribing to channel_joined/left events.
```

Comment:
The framework should make sure these subscriptions are removed when the bot is removed from this team cc @levb for discussion